apiserver/apiserver.go

```diff
@@ -58,6 +59,11 @@ var logger = loggo.GetLogger("juju.apiserver")
 
 var defaultHTTPMethods = []string{"GET", "POST", "HEAD", "PUT", "DELETE", "OPTIONS"}
 
+// SQLDBGetter provides access to model-scoped SQL databases.
+type SQLDBGetter interface {
```

Comment:
I prefer nouns over {verb}-er. `DBFactory` would by my choice.

Reply:
No particular preference here; just wanted to use the idiomatic naming conventions for interfaces (io.Reader/Writer etc.).

In this case in particular, perhaps a `DBResolver` would be a better name compared to the term "factory"